### PR TITLE
Export from VSCode extension + crypted Python DataPlugins

### PR DESCRIPTION
**Pull request by jschumme**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/systemlink-operations-handbook/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

- Explanation + screenshot: How to export a DataPlugin with VSCode extension
- Explanation: The possibility to export crypted DataPlugins in DIAdem

### What testing has been done?

- Linting done upfront and ensured by auto tests

### CODE REVIEW

File: handbook/python-dataplugins/python-dataplugins.md [hunk 1]
```diff
@@ -285,6 +285,20 @@ export a DataPlugin as a URI file.
...
 
+You can also export a DataPlugin from NI's [Python DataPlugin Extension](https://github.com/ni/vscode-ni-python-dataplugins)
+for Visual Studio Code:
+
+<figure>
+   <img src="../../img/pydp-vscode_export.png" width="600" />
+   <figcaption>Exporting DataPlugins in VSCode</figcaption>
+</figure>
+
+### Crypted DataPlugins
```
Comment (GrKuehnH) [inline handbook/python-dataplugins/python-dataplugins.md:296]: Encrypted?

File: handbook/python-dataplugins/python-dataplugins.md [hunk 2]
```diff
@@ -285,6 +285,20 @@ export a DataPlugin as a URI file.
...
+
+In some cases, you may want to protect your code from being accessed by others.
+Use DIAdem to export a crypted Python DataPlugin, which protects your source
```
Comment (GrKuehnH) [inline handbook/python-dataplugins/python-dataplugins.md:299]: encrypted?

File: handbook/python-dataplugins/python-dataplugins.md [hunk 3]
```diff
@@ -285,12 +285,26 @@ export a DataPlugin as a URI file.
...
+
+<figure>
+   <img src="../../img/pydp-vscode_export.png" width="600" />
+   <figcaption>Exporting DataPlugins in VSCode</figcaption>
+</figure>
+
+### Encrypted DataPlugins
+
+In some cases, you may want to protect your code from being viewed by others.
+Use DIAdem to export an encrypted Python DataPlugin. This functionality is not
```
Comment (JD-Robertson) [inline handbook/python-dataplugins/python-dataplugins.md:299]: Should this be "NI DIAdem"?

File: handbook/python-dataplugins/python-dataplugins.md [hunk 4]
```diff
@@ -285,12 +285,26 @@ export a DataPlugin as a URI file.
...
+available in the VSCode extension.
+
 ## Known Limitations
 
 ### NumPy and Pandas
 
-Unfortunately, NumPy and Pandas are not well supported to run in embedded Python
-environments and, therefore, cannot be used in DataPlugins.
+NumPy and Pandas do not run reliably in embedded Python environments and, therefore,
+we recommend not to use them in DataPlugins.
```
Comment (JD-Robertson) [inline handbook/python-dataplugins/python-dataplugins.md:307]: "NI does not recommend them for use in DataPlugins"

**Review by JD-Robertson — APPROVED** [review]

File: handbook/python-dataplugins/python-dataplugins.md (hunk 3 above)
Comment (jschumme) [reply handbook/python-dataplugins/python-dataplugins.md:299]: I believe you can let DIAdem stay alone, see https://www.ni.com/en-us/shop/software/products/diadem.html
@GrKuehnH can you confirm?

**Review by cameronwaterman — APPROVED** [review]

**Pull request merged**